[2024-04-17T16:05:28.672+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-17T16:05:28.681+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-17T16:05:28.682+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-17T16:05:28.695+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data_from_text> on 2024-04-16 00:00:00+00:00
[2024-04-17T16:05:28.701+0000] {standard_task_runner.py:60} INFO - Started process 77 to run task
[2024-04-17T16:05:28.704+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'extract_data_from_text', 'scheduled__2024-04-16T00:00:00+00:00', '--job-id', '2034', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmpn_3xmr7d']
[2024-04-17T16:05:28.706+0000] {standard_task_runner.py:88} INFO - Job 2034: Subtask extract_data_from_text
[2024-04-17T16:05:28.748+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_data.extract_data_from_text scheduled__2024-04-16T00:00:00+00:00 [running]> on host fdd3cfdf3f53
[2024-04-17T16:05:28.822+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='extract_data_from_text' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-16T00:00:00+00:00'
[2024-04-17T16:05:28.823+0000] {python.py:202} INFO - Done. Returned value was: file does not exist
[2024-04-17T16:05:29.025+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=extract_data_from_text, execution_date=20240416T000000, start_date=20240417T160528, end_date=20240417T160529
[2024-04-17T16:05:29.078+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-17T16:05:29.114+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
